exalearn-ui/src/app/components/new-grammar/new-grammar.component.ts

```diff
@@ -38,7 +44,7 @@ export class NewGrammarComponent implements OnInit {
 	}
 
 	submit(): void {
-		this.ncService.setIsCorrectProperty(this.form);
+		/* this.ncService.setIsCorrectProperty(this.form);
```

Comment:
Do we really need to store this comment? It's better to remove all unnecessary comments